lib/Exception/WebDriverException.php

```diff
@@ -83,8 +84,56 @@ public function getResults()
      * @throws WebDriverCurlException
      * @throws XPathLookupException
      */
-    public static function throwException($status_code, $message, $results)
+    public static function throwException($status_code, $message, $results, $w3c_compliant = false)
```

Comment:
Wouldn't be possible to use polymorphism here and rather have the method implemented as part of something like W3C Compliant Exception?

Reply:
you don't even need this. Identifying the W3C vs JsonWire status codes is easy, as JsonWire uses numbers, while W3C uses non-numeric strings.